packages/manager/modules/pci/src/projects/project/sidebar/sidebar.constant.js

```diff
@@ -79,6 +79,13 @@ export const MENU = [
         },
         translation: 'cloud_sidebar_orchestration_kubernetes',
       },
+      {
+        id: 'private-registry',
+        options: {
+          state: 'pci.projects.project.private-registry',
+        },
+        translation: 'cloud_sidebar_orchestration_private_registry',
```

Comment:
Translation not added in this PR because already present in translations files (it's ok 👍 )